pkgs/tools/networking/goodhosts/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub, testers }:
+
+buildGoModule rec {
+  pname = "goodhosts";
+  version = "v1.1.0";
+  name = "goodhosts-${version}";
+
+  src = fetchFromGitHub {
+    owner = "goodhosts";
+    repo = "cli";
+    rev = version;
+    sha256 = "sha256-BsStTOuMercplx+g+egjiBjCKf5UFCDLGqGVo6i5yKQ=";
+  };
+
+  ldflags = [
+    "-s -w -X main.version=${version}"
+  ];
+
+  postInstall = ''
+    mv $out/bin/cli $out/bin/goodhosts
+  '';
+
+
+  vendorHash = null;
+
+  meta = with lib; {
+    description = "A CLI tool for managing hostfiles";
+    license = licenses.mit;
+    homepage = "https://github.com/goodhosts/cli/tree/main";
+    maintainers = with maintainers; [ schinmai-akamai ];
+  };
```

This can be improved further to conform to the [pkgs/CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes) by adding a `mainProgram` meta attribute.

```suggestion
    maintainers = with maintainers; [ schinmai-akamai ];
    mainProgram = "goodhosts";
  };
```